Modules/Revenue/Database/Migrations/2022_09_09_171756_create_revenue_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateRevenueTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('Revenue_proceeds', function (Blueprint $table) {
```

Comment:
```suggestion
        Schema::create('revenue_proceeds', function (Blueprint $table) {
```

We never use the capital letters in the table name